Website/src/rendering/src/assets/css/components/Products/ProductDetails.css

```diff
@@ -54,6 +66,12 @@
         }
       }
 
+      &-overview {
+        @apply col-span-1 -mx-[20px];
```

Comment:
Is `-mx-[20px]` there to counter `.shop-container` `mx-5`? Can we instead use `-mx-5` for that to keep using REMs?